.github/workflows/integration-e2e.yml

```diff
@@ -0,0 +1,26 @@
+name: integration-e2e
+on:
+  push:
+    branches:
+    - '**'
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Build IBC Simulator BCD Image
+        run: make build-ibcsim-bcd
+
+      - name: Build Babylond Image
+        run: make build-babylond
+
+      - name: Cache Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.23
+
+      - name: Run Integration Tests
+        run: sudo make test-e2e-bcd-consumer-integration
```

Comment:
Better to create a comprehensive `test-e2e` target in the Makefile, and call it from here. I'm pushing a companion / follow-up PR with some improvements, if you don't mind.